lib/results_validators/persons_validator.rb

```diff
@@ -7,9 +7,9 @@ class PersonsValidator < GenericValidator
     WHITESPACE_IN_NAME_ERROR = "Person '%{name}' has leading/trailing whitespaces or double whitespaces."
     WRONG_WCA_ID_ERROR = "Person %{name} has a WCA ID which does not exist: %{wca_id}."
     WRONG_PARENTHESIS_FORMAT_ERROR = "Opening parenthesis in '%{name}' must be preceded by a space."
-    DOB_0101_WARNING = "The date of birth of %{name} is on January 1st, please ensure it's correct."
-    VERY_YOUNG_PERSON_WARNING = "%{name} seems to be less than 3 years old, please ensure it's correct."
-    NOT_SO_YOUNG_PERSON_WARNING = "%{name} seems to be around 100 years old, please ensure it's correct."
+    DOB_JAN_ONE = "dob_jan_one"
+    DOB_TOO_YOUNG = "dob_too_young"
+    DOB_TOO_OLD = "dob_too_old"
```

Comment:
Can be changed to symbol: `:dob_jan_one` etc

Reply:
Done